htlcswitch/link_test.go

```diff
@@ -3910,15 +3919,6 @@ func TestChannelLinkAcceptDuplicatePayment(t *testing.T) {
 	case <-time.After(5 * time.Second):
 		t.Fatalf("payment result did not arrive")
 	}
-
-	// Now, if we attempt to send the payment *again* it should be rejected
-	// as it's a duplicate request.
-	err = n.aliceServer.htlcSwitch.SendHTLC(
```

Comment:
Reminder to make sure that this test is replicated at the router level. It probably already is, but just dropping a not here to ensure we don't forget.

Reply:
Tested on the router level here: https://github.com/lightningnetwork/lnd/pull/2761/commits/1bab8fc2fdfb9a371c2c7a4a25e943534b7acde4#diff-228fed2395e7c9a4ddf89a53ca461892R2646